dokku

```diff
@@ -70,6 +70,9 @@ case "$1" in
     DOKKU_SCALE_FILE="$DOKKU_ROOT/$APP/DOKKU_SCALE"
     oldids=$(get_container_ids $APP)
 
+    [[ -f "$DOKKU_ROOT/ENV" ]] && source $DOKKU_ROOT/ENV
+    [[ -f "$DOKKU_ROOT/$APP/ENV" ]] && source $DOKKU_ROOT/$APP/ENV
```

Comment:
these lines may have wider consequences than intended. we've seen in the past where we've added sourcing of app envs cause some issues. that isn't to say these line should be changed but that we may want to remember them specifically when troubleshooting after this gets merged. cc: @josegonzalez